source/scriptHandler.py

```diff
@@ -324,6 +324,13 @@ def clearLastScript():
 	_lastScriptCount = 0
 
 
+def getCurrentScript():

```

Comment:
can this please get a return type added

Reply:
Fixed in 4a8a7db9ad5a670f9eb4a62e3392e3cd36a1d050